app/src/main/java/com/architectcoders/aacboard/data/PictogramUI.kt

```diff
@@ -0,0 +1,13 @@
+package com.architectcoders.aacboard.data
```

Comment:
Wrong packege? UI models should be in the ui package not in data package

Reply:
Which package name and location do you propose? Antonio's project does not require UI models classes, so I am not sure which one to use: 
/main/java/com/architectcoders/aacboard/data/ui/PictogramUI.kt ?
/main/java/com/architectcoders/aacboard/ui/data/PictogramUI.kt ?
/main/java/com/architectcoders/aacboard/ui/model/PictogramUI.kt ?
other?

Reply:
I would go for this one: `/main/java/com/architectcoders/aacboard/ui/model/PictogramUI.kt` or you can also put it inside the viewmodel who is using this class, `SearchPictogramsViewModel` if I am not mistaken

Reply:
Moved to /main/java/com/architectcoders/aacboard/ui/model/PictogramUI.kt